web-analogs.md

```diff
@@ -9,16 +9,42 @@ permalink: /web-analogs/
 {:toc}
 
 <div class="begin-examples"></div>
-This page is for users who are familiar with the HTML and CSS syntax for arranging components of an application's UI. It maps HTML/CSS code snippets to their Flutter/Dart code equivalents.
+This page is for users who are familiar with the HTML and CSS syntax for 
+arranging components of an application's UI. It maps HTML/CSS code snippets to 
+their Flutter/Dart code equivalents.
+
+The examples assume:
+* The HTML doctype is set to HTML5, and the CSS box model for all HTML elements 
```

Comment:
"HTML5" is an obsolete buzzword, I would go with just "The HTML document starts with a modern HTML DOCTYPE, ..."